### PR TITLE
Toast: Multiline with button & fix for ripple effect

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1303
[Problem] Multiline toast message looks wrong
[Solution]
 - added example of multiline toast with button
 - added rounding for ripple effect

[Demo]
toast-multi

toast-single

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

**Comment by TomaszLukawskiSam** [comment]
@lslachciak 
There is problem with CI (spell check). Could you advise me what to do?

**Comment by lslachciak** [comment]
> 
> 
> @lslachciak
> There is problem with CI (spell check). Could you advise me what to do?

This worked for me:
1) Update csPell.json
```
diff --git a/cSpell.json b/cSpell.json
index 08b20e9e..e5887a23 100644
--- a/cSpell.json
+++ b/cSpell.json
@@ -938,12 +938,12 @@
         "lorem",
         "ipsum",
         "amet",
+        "adipisicing",
         "invenire",
         "moderatius",


```
2) fix typo
```
diff --git a/examples/mobile/UIComponents/components/toast/index.html b/examples/mobile/UIComponents/components/toast/index.html
index d32ffbfe..fca6aeb6 100644
--- a/examples/mobile/UIComponents/components/toast/index.html
+++ b/examples/mobile/UIComponents/components/toast/index.html
@@ -60,7 +60,7 @@
                        </div>
                        <div class="ui-popup ui-popup-toast ui-popup-toast-multiline" data-role="popup" id="popup_multiline_btn_toast">
                                <div class="ui-popup-content ui-popup-toast-has-button">
-                                       Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmo.
+                                       Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod.
                                        <button class="ui-btn">
                                                Action
                                        </button>

```

File: examples/mobile/UIComponents/components/toast/index.html [hunk 1]
```diff
@@ -53,6 +58,14 @@ <h1>
 					</button>
 				</div>
 			</div>
+			<div class="ui-popup ui-popup-toast ui-popup-toast-multiline" data-role="popup" id="popup_multiline_btn_toast">
+				<div class="ui-popup-content ui-popup-toast-has-button">
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmo.
```
Comment (lslachciak) [inline examples/mobile/UIComponents/components/toast/index.html]: euismod is a correct word.
Don't you know latin? ;)

**Review by lslachciak — COMMENTED** [review]
works fine, expect some spell issues,
as a part of issue https://github.com/Samsung/TAU/issues/1303
could you also add an example (may be in separate PR) to show multiline toast withouth button? I want to make sure that it works fine also in this case

File: examples/mobile/UIComponents/components/toast/index.html (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/toast/index.html]: :)

**Review by lslachciak — APPROVED** [review]
works fine, looks great!

File: examples/mobile/UIComponents/components/toast/index.html [hunk 2]
```diff
@@ -53,6 +63,19 @@ <h1>
 					</button>
 				</div>
 			</div>
+			<div class="ui-popup ui-popup-toast ui-popup-toast-multiline" data-role="popup" id="popup_multiline_toast">
+				<div class="ui-popup-content">
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do euismod.
```
Comment (lslachciak) [inline examples/mobile/UIComponents/components/toast/index.html]: eiusmod not euismod
see https://en.wikipedia.org/wiki/Lorem_ipsum

File: cSpell.json [hunk 3]
```diff
@@ -938,13 +938,15 @@
...
         "amet",
+        "adipisicing",
         "invenire",
         "moderatius",
         "mucius",
         "consectetur",
         "adipiscing",
         "elit",
         "eiusmod",
+        "euismod",
```
Comment (lslachciak) [inline cSpell.json]: not needed change

**Review by lslachciak — COMMENTED** [review]
need to fix one more

**Review by lslachciak — APPROVED** [review]

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**